src/acl/Gadgets.cc

```diff
@@ -79,9 +79,7 @@ aclIsProxyAuth(const char *name)
 
     debugs(28, 5, "aclIsProxyAuth: called for " << name);
 
-    ACL *a;
-
-    if ((a = ACL::FindByName(name))) {
+    if (auto a = Acl::ACL::FindByName(name)) {
```

Comment:
```suggestion
    if (const auto a = Acl::ACL::FindByName(name)) {
```

I normally do not ask you to fix `const` in such contexts, but since moving `a` declaration is itself a change outside this PR scope, I think it is best to finish this out-of-scope polishing started by this PR in this PR. Same for similar `const` requests in this review. As usual, if some of them do not compile, then just paste a compiler error message to address them.